### PR TITLE
Fix typo for "Add FEATURE_LOR for AArch64 v8.1"

### PR DESCRIPTION
**Pull request by kuhanov**
Looks like typo in 'if' condition. Extend threshold to AA64MMFR1

### CODE REVIEW

**Comment by AssadHashmi** [comment]
Added @kuhanov to AArch64 Jenkins CI user list.

**Comment by kuhanov** [comment]
> run arm tests
Hi.
Did you mean 'ci-test-suite-aarchxx' action workflow?

File: core/arch/aarch64/proc.c [hunk 1]
```diff
@@ -182,7 +182,7 @@ proc_has_feature(feature_bit_t f)
     uint64 freg_val = 0;
 
     feature_reg_idx_t feat_reg = GET_FEAT_REG(f);
-    if (feat_reg >= AA64ISAR0 && feat_reg <= AA64PFR0) {
+    if (feat_reg >= AA64ISAR0 && feat_reg <= AA64MMFR1) {
```
Comment (AssadHashmi) [inline core/arch/aarch64/proc.c:185]: Thank you for finding this issue and raising a fix PR.

`AA64MMFR1` should be replaced with `AA64DFR0`, see `feature_reg_idx_t` in `proc_api.h`, https://github.com/DynamoRIO/dynamorio/blob/d941df9556f1e0b53d406ce59b17d444f1271182/core/arch/proc_api.h#L188

Also, `AA64PFR0` needs to be replaced with `AA64DFR0' in `proc_has_feature()`, see https://github.com/DynamoRIO/dynamorio/blob/d941df9556f1e0b53d406ce59b17d444f1271182/core/arch/aarch64/proc.c#L185

And a new `case` clause is required in `proc_has_feature()` for `AA64DFR0` after `case AA64MMFR1`, see https://github.com/DynamoRIO/dynamorio/blob/d941df9556f1e0b53d406ce59b17d444f1271182/core/arch/aarch64/proc.c#L202
```

**Review by AssadHashmi — CHANGES_REQUESTED** [review]

**Comment by AssadHashmi** [comment]
> run arm tests
> Hi.
> Did you mean 'ci-test-suite-aarchxx' action workflow?

No, the `run arm tests` comment triggers the `AArch64 Precommit` tests on our Jenkins instance. It's a manual trigger. I have added you to the Jenkins user list now and all future commits by you should automatically trigger the `AArch64 Precommit` tests.

File: core/arch/aarch64/proc.c (hunk 1 above)
Comment (kuhanov) [reply core/arch/aarch64/proc.c]: Ok. Done.

**Comment by AssadHashmi** [comment]
run arm tests

**Review by AssadHashmi — APPROVED** [review]

**Pull request merged**